smart_proxy_container_gateway.gemspec

```
@@ -5,18 +5,18 @@ Gem::Specification.new do |s|
   s.version = Proxy::ContainerGateway::VERSION
 
   s.summary = 'Pulp 3 container registry support for Foreman/Katello Smart-Proxy'
-  s.description = 'Pulp 3 container registry support for Foreman/Katello Smart-Proxy'
+  s.description = 'A plug-in for Pulp 3 container registry support for Foreman/Katello Smart-Proxy'
```

I'd look at @ianballou here and wonder if this is better. I know you change this because the strict verification doesn't like a summary that equals the description.
```suggestion
  s.description = 'Foreman Smart Proxy plug-in for Pulp 3 container registry support'
```

---

I like the sound of this :+1: